app/controllers/state_file/intake_logins_controller.rb

```diff
@@ -1,7 +1,6 @@
 module StateFile
   class IntakeLoginsController < Portal::ClientLoginsController
     helper_method :prev_path, :illustration_path
-    before_action :redirect_to_data_review_if_intake_authenticated
```

Comment:
We remove this because it means the sign in page is skipped if a user is already logged in (By for example visiting the landing page in the last 15 minutes)